postman/response.json

```diff
@@ -0,0 +1 @@
+{"id":9222968140497191874,"username":"Mallory","firstName":"Luisa","lastName":"Emmett Stokes","email":"Cullen_Schamberger68@yahoo.com","password":"12345","phone":"string","userStatus":0}
```

Comment:
Asi nerozumiem co sa tu deje 😄 Je to export JSONu s unikatnymi datami? 
Predstavoval som ti skor zmenu v https://github.com/FilipRechtoris/git_workshop_wta13B/blob/main/postman/PETSTORE.postman_collection.json ako napriklad pridanie testu na niektory z REST requestov.